tools/backup/calllogs_export.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should these files be called `_example.xml`? So people know that they need to put their own files here?